assets/js/base/components/price-slider/style.scss

```
@@ -180,14 +180,23 @@
 		}
 	}
 
-	&.is-loading {
+	&.is-loading,
+	&.is-disabled {
 		.wc-block-price-filter__range-input-wrapper,
 		.wc-block-price-filter__amount,
 		.wc-block-price-filter__button {
 			@include placeholder();
 			box-shadow: none;
 		}
 	}
+
+	&.is-disabled:not(&.isLoading) {
```

You can remove `&` from the `not` selector. It should be equivalent and with less specificity. Also, `isLoading` should be `is-loading`, no?
```suggestion
	&.is-disabled:not(.is-loading) {
```